tests/layer_tests/pytorch_tests/test_tuple_construct.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import numpy as np
+from pytorch_layer_test_class import PytorchLayerTest
+
+
+class TestTupleConstruct(PytorchLayerTest):
+    def _prepare_input(self):
+        return (np.random.uniform(0, 50, (1, 10)).astype(np.float32),)
+
+    def create_model(self, case):
+        import torch
+
+        class prim_tuple_construct_single_value(torch.nn.Module):
+
+            def forward(self, x):
+                return (x,)
+
+        class prim_tuple_construct(torch.nn.Module):
+
+            def forward(self, x):
```

Comment:
Should we test tuples in parameters to `forward`?

Reply:
We test output for now